.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I personally prefer using the YAML syntax for this file, I feel it strongly improves readability. [Example](https://github.com/sgmap/mes-aides-ui/blob/rewrite/.eslintrc).

WDYT?